### PR TITLE
fix: update pvc size

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

PVC size cannot be updated now, this PR will fix the problem.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle @cd1989 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/server/biz/utils/retry.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package utils
```
Comment (supereagle) [inline pkg/server/biz/utils/retry.go]: This package can be used by other components, what about put it into `pkg/util` package.

File: pkg/server/handler/v1alpha1/integration.go [hunk 2]
```diff
@@ -349,43 +339,59 @@ func getIntegration(tenant, name string) (*api.Integration, error) {
...
+
+	oldCluster := oldIn.Spec.Cluster
+	cluster := in.Spec.Cluster
+	if cluster.Namespace == "" {
+		cluster.Namespace = oldCluster.Namespace
+	} else if cluster.Namespace == oldCluster.Namespace && cluster.PVC == "" {
+		cluster.PVC = oldCluster.PVC
+	}
+	// turn on worker cluster
+	if !oldCluster.IsWorkerCluster && cluster.IsWorkerCluster {
```
Comment (supereagle) [inline pkg/server/handler/v1alpha1/integration.go]: Handle these four cases by two-layer nesting if-else?

File: pkg/server/handler/v1alpha1/integration.go [hunk 3]
```diff
@@ -349,43 +339,59 @@ func getIntegration(tenant, name string) (*api.Integration, error) {
...
-		// turn on worker cluster
-		if !oldCluster.IsWorkerCluster && cluster.IsWorkerCluster {
-			// open cluster for the tenant, create namespace and pvc
-			err := OpenClusterForTenant(in, tenant)
-			if err != nil {
-				return nil, err
-			}
+	}
+
+	// TODO(zhujian7): namespace or pvc changed
```
Comment (supereagle) [inline pkg/server/handler/v1alpha1/integration.go]: Remove todo?

File: pkg/server/handler/v1alpha1/integration.go (hunk 2 above)
Comment (supereagle) [inline pkg/server/handler/v1alpha1/integration.go]: Handle these four cases by two-layer nesting if-else?

File: pkg/server/handler/v1alpha1/tenant.go [hunk 4]
```diff
@@ -164,16 +170,23 @@ func UpdateTenant(ctx context.Context, name string, newTenant *api.Tenant) (*api
...
+				continue
+			}
+
 			client, err := common.NewClusterClient(&cluster.Credential, cluster.IsControlCluster)
 			if err != nil {
 				log.Warningf("new cluster client for integration %s error %v", integration.Name, err)
 				continue
 			}
 
+			log.Info("old PersistentVolumeClaim: %+v ", tenant.Spec.PersistentVolumeClaim)
```
Comment (supereagle) [inline pkg/server/handler/v1alpha1/tenant.go]: log.Infof()

Comment (zhujian7) [reply pkg/server/handler/v1alpha1/tenant.go]: ok

File: pkg/server/biz/utils/retry.go (hunk 1 above)
Comment (zhujian7) [reply pkg/server/biz/utils/retry.go]: ok

File: pkg/server/handler/v1alpha1/integration.go (hunk 3 above)
Comment (zhujian7) [reply pkg/server/handler/v1alpha1/integration.go]: no, I just logged something

File: pkg/server/handler/v1alpha1/integration.go (hunk 2 above)
Comment (zhujian7) [reply pkg/server/handler/v1alpha1/integration.go]: I want to keep the current form since it is easy to understand.

**Comment by zhujian7** [comment]
@supereagle PTAL

**Pull request merged**